homework.md

```diff
@@ -44,70 +44,71 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used to add functionality to websites and web applications
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+When people want to collaborate easily. Can always revert back to the old version of the project,
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+When you fork a repository, you create a copy of the original repository (upstream repository) to your GitHub but the repository remains on your GitHub account.
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+You copy the repo on to your computer
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+Pull requests let you tell others about changes you've pushed to a GitHub repository. Once a pull request is sent, interested parties can review the set of changes, discuss potential modifications, and even push follow-up commits if necessary.
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+By default, with no arguments, git log lists the commits made in that repository in reverse chronological order; that is, the most recent commits show up first. As you can see, this command lists each commit with its SHA-1 checksum, the author's name and email, the date written, and the commit message.
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+The git push command is used to upload local repository content to a remote repository. Pushing is how you transfer commits from your local repository to a remote repo. It's the counterpart to git fetch , but whereas fetching imports commits to local branches, pushing exports commits to remote branches
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+GitHub, Inc. is an American multinational corporation that provides hosting for software development and version control using Git. It offers the distributed version control and source code management functionality of Git, plus its own features.
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+The "commit" command is used to save your changes to the local repository. ... Using the "git commit" command only saves a new commit object in the local Git repository.
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Visual Studio Code
 
 ## Question 17
 
 What do you use "Commit Messages" for?
 
-To record what changes you made to the code
+To show other developers what you have changed and why. This helps to stop confusion.
 
 ## Question 18
 
 Where can I look to find help with Git?
 
 The Git Cheatsheet!
+
```

Comment:
Maybe look into adding another suggestion to find help with GIT?

Reply:
Thanks will do